src/app/app.config.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * SPDX-FileCopyrightText: (c) 2023 Carl Zeiss AG
+ * SPDX-License-Identifier: MIT
+ */
+
+import { ApplicationConfig, importProvidersFrom } from '@angular/core';
+import { provideRouter } from '@angular/router';
+
+import { routes } from './app.routes';
```

Comment:
In some places imports are done with `@app/<something>` and sometimes with `./<something>`.

Shouldn't we try to have a common style here?

Maybe this is a topic for a different PR and I'm not sure what's best practice in realworld apps.

Reply:
My understanding for the imports path handling is (and the tooling seems to work the same way):
* when the imported file is in the same directory, use "from './foo.bar'". 
* when importing from other directories, try to use the defined @ shortcuts

Most of the project files are importing this way, despite a few exceptions (not part of this PR). Therefore I would suggest to address this in a separate branch, if needed. Feel free to reopen this PR issue if you see it differently.